stargaze/assetlist.json

```diff
@@ -632,6 +632,41 @@
         }
       ],
       "type_asset": "sdk.coin"
+    },
+    {
+      "description": "GATA Yield DAO Governance Token",
+      "denom_units": [
+        {
+          "denom": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
+          "exponent": 0
+        },
+        {
+          "denom": "ygata",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/omniflix1fwphj5p6qd8gtkehkzfgac38eur4uqzgz97uwvf6hsc0vjl004gqfj0xnv/ygata",
```

Comment:
This denom cannot exist on Stargaze. I assume this is actually an ICS20 asset, IBC-transferred from Omniflix?

Reply:
yes exactly, will resubmit, thank you.